packages/iov-cli/README.md

```diff
@@ -147,43 +147,45 @@ UserProfile {
   ...
 ```
 
-### Register a BNS name
+### Register a BNS username NFT
 
 Assuming you have a `profile`, a `signer` and a `recipient` identity with
 transactions associated from above
 
 ```
 > .editor
-const setNameTx: SetNameTx = {
-  kind: "bns/set_name",
+const registration: RegisterUsernameTx = {
```

Comment:
Maybe add a TODO for actually showing the chain/address pairs in use to associate it
(I know, right now it needs the ugly check/create blockchain workaround, but we should clean that up and update this soon-ish)